packages/expandable-search-bar/test/promised-sleep.js

```diff
@@ -0,0 +1,3 @@
+export default function promisedSleep(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
```

Comment:
I didn't see this get imported in. should we remove?
also, what is this function for, primarily?

Reply:
Removed. It's used to sleep execution without blocking the thread. I typically use it when I test calling some method on one of elements where the update is going to be asynchronous, but I need to wait for it to update to check the results.

I was trying to find a solution through LitElement, like some event to let me know that the update has happened, but I couldn't find anything that didn't seem super hacky.